go.mod

```diff
@@ -93,10 +96,10 @@ require (
 	go.opentelemetry.io/otel/sdk/metric v1.31.0
 	go.opentelemetry.io/otel/trace v1.31.0
 	go.opentelemetry.io/proto/otlp v1.3.1
-	golang.org/x/crypto v0.31.0
+	golang.org/x/crypto v0.32.0
 	golang.org/x/exp v0.0.0-20241108190413-2d47ceb2692f
 	golang.org/x/mod v0.22.0
-	golang.org/x/net v0.33.0
+	golang.org/x/net v0.34.0
```

Comment:
As we have already cut v0.20 with v1.16 can you leave it at that for now so we don't need to bump unrelated dependencies like `x/net` again. You can update it later in master branch.

Reply:
Seems to be a peer dep coming from `github.com/Azure/azure-sdk-for-go/sdk/azcore`: https://github.com/Azure/azure-sdk-for-go/blob/sdk/azcore/v1.17.0/sdk/azcore/go.mod#L8